src/widget/text-print/text-print.js

```diff
@@ -22,27 +22,25 @@ class TextPrintWidget extends Widget {
     }
 
     _addWidget() {
-        const className = 'print-text-input';
```

Comment:
Note, you had swapped print-input-text with print-text-input. ;) This broke the recent CSS fixes.